lib/KDropdownmenu/__tests__/ KDropdownMenu.spec.js

```diff
@@ -0,0 +1,29 @@
+// eslint-disable-next-line import/named
+import { renderComponent, takeSnapshot, click } from '../../../jest.conf/visual.testUtils';
```

Comment:
Was there any problem with having these tests as you were implementing them in https://github.com/learningequality/kolibri-design-system/pull/933? A test case for each snapshot.

Sorry if there was any miscomunication on my side :sweat_smile:. I meant that we could use the custom test component just for the last test case, the `KDropdownMenu - With Header` snapshot.